src/server.cpp

```diff
@@ -2045,6 +2055,9 @@ s32 Server::playSound(const SimpleSoundSpec &spec,
 			RemotePlayer *player = m_env->getPlayer(client_id);
 			if (!player)
 				continue;
+			if (!params.exclude_player.empty() &&
+				params.exclude_player.compare(player->getName()) == 0)
```

Comment:
+1 indent.
Why not `if (params.exclude_player == player->getName())`? Empty names are not possible for players.

Reply:
empty name is possible if no players are excluded

Reply:
`exclude_player` is an std::string, `player->getName()` a const char*
not sure if == does the right thing with those two